### ISSUE TITLE
Desktop integration of the Linuxversion  is broken

### ISSUE THREAD

**Issue by amogus07** [kukuruzka165/materialgram#4]
### Steps to reproduce

Run `sudo ./install.sh` from the root of the extracted release archive

### Expected behaviour

materialgram appears in the app grid

### Actual behaviour

materialgram doesn't appear in the app grid, even after a rebooting, but Main Menu (Libre Menu Editor) detects the entry. When launching materialgram manually from the terminal, the icon in the dash representing it is empty/broken.

### Operating system

Fedora 39 (Workstation Edition)

### Version of Telegram Desktop

4.12.2.1

### Installation source

Static binary from official website

**Comment by samjaredvscode** [comment]
use flatpak bro, is better

**Comment by amogus07** [comment]
@samjaredvscode I use the flatpak now, I opened the issue before it was published. But it's still relevant for people who don't want to use flatpak.

**Comment by samjaredvscode** [comment]
You're right about that, my bad. Remember that the alternatives that the developers give us are for the effective use of the application.
